repositories/OptionRepository.js

```diff
@@ -71,3 +71,16 @@ module.exports.remove = function remove(id) {
 module.exports.undelete = function(id) {
   return Option.update(id, { isDeleted: false }).then(head);
 };
+
+module.exports.getOtherOption = function(
+  id,
+  where = {},
+  orderBy = "created_at",
+  direction = "asc"
+) {
+  return Option.findAll()
+    .where({ isDeleted: false, otherAnswerId: id })
```

Comment:
Why don't we just make the default value = `{ isDeleted: false, otherAnswerId: id }` then if we want to use a different query for any reason we can.

Reply:
After thinking about it some more, I'm not sure we should do this. I can't think of a case where we'd ever want to retrieve the deleted other option, and we'd always want to get the Other entry for a specific answer Id.

Equally, I don't think we'd ever want to specialise this query any further. So it might make more sense to remove the second where clause altogether.